src/Admin/Options.php

```diff
@@ -595,6 +595,26 @@ function () {
 			'sf_feed_settings_categories'
 		);
 
+		// Show 0 stock varitions in feed
+		add_settings_field(
+			'zero_stock_variations',
+			__( 'Show variations with stock 0', 'shopping-feed' ),
```

Comment:
Je trouve le wording bizarre. Je changerais pour quelque chose comme "Include out of stock variations in the feed".

Il faudrait revoir les autres endroits où on utilise "zero_stock" pour "out_of_stock"